components/scripts/gradle/03-validate-local-build-caching-different-locations.sh

```diff
@@ -47,6 +55,12 @@ main() {
   exit_with_return_code
 }
 
+verify_scans_support_tools() {
```

Comment:
I suggest to rename to `verify_build_scan_support_tool_exists`.

Reply:
Changed in 959de12707c38c55194c27955284d8a8bc5830ca